Pipfile.lock

```diff
@@ -24,12 +24,13 @@
             "index": "pypi",
             "version": "==20.3.0"
         },
-        "frozendict": {
+        "immutabledict": {
```

Comment:
Found this issue while running tests locally. Python throws out a warning because `frozendict` imports abstract classes from `collections`. Since `frozendict` is no longer maintained, moving to `immutabledict`.